### PR TITLE
Hotfix/redemption error

### LINKED ISSUE #169: Remove `?redeem=`  query parameter after successful redemption
Whenever you redeem a redemption code using something like `http://localhost:3000/?redeem=<KEY>`, it will not remove the `?redeem=<KEY>` if it was successful. That means that every time the user refreshes the page it will display the error 'GraphQL error: Redemption code has expired or already been redeemed.'

### PR DESCRIPTION
**Pull request by ClayMav**
# Pull Request Description
Simply redirects to the / after you redeem your code, stopping error from occurring on refresh
Closes #169 

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by ClayMav** [comment]
Currently absorbing the #182 PR as well, as it is required for me to run the code.

File: apps/web/src/screens/HomePage/index.tsx [hunk 1]
```diff
@@ -84,10 +88,13 @@ const HomePage: React.FC<{}> = () => {
...
             <Toast
               iconName={"alert-circle-outline"}
-              message={"Success! Redemption code applied to your account. Please refresh to see changes."}
+              message={
+                "Success! Redemption code applied to your account. Please refresh to see changes."
+              }
               fill={"green"}
             />
           );
+          props.history.push("/");
```
Comment (kevinschoonover) [inline apps/web/src/screens/HomePage/index.tsx]: There may be other query parameters that we don't want to squash

Comment (ClayMav) [reply apps/web/src/screens/HomePage/index.tsx]: Disagree, if they are doing a redemption code that should their only parameter.

**Comment by ClayMav** [comment]
As I mentioned in my previous comment, I cannot develop on master since docker is broken in develop, so until that PR is accepted I make all my branches on that PR

File: apps/web/src/screens/HomePage/index.tsx (hunk 1 above)
Comment (kevinschoonover) [reply apps/web/src/screens/HomePage/index.tsx]: Not necessarily. For example, auth0 uses the `code` query parameter for login which will squash. The argument can be made that they will be logged in before running this, but future libraries can use this behavior for other things and we will then transparently fail

Comment (ClayMav) [reply apps/web/src/screens/HomePage/index.tsx]: On further review, I agree with you, though I am not sure we will ever see this problem, authentication is handled earlier than redemption code stuff. I will fix this

File: apps/web/src/screens/HomePage/index.tsx [hunk 2]
```diff
@@ -35,7 +35,11 @@ const NavHeroWrapper = styled.div`
   }
 `;
 
-const HomePage: React.FC<{}> = () => {
+interface IHomePageProps {
+  history: any;
```
Comment (kevinschoonover) [inline apps/web/src/screens/HomePage/index.tsx]: `any`

**Pull request merged**